uint32 id
---
bool status

